functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/affected/LocalChangesAffectedTestsSelectionExecutionFunctionalTest.java

```diff
@@ -30,16 +35,17 @@ public void should_only_execute_tests_related_to_single_local_change_in_producti
     }
 
     @Test
-    public void should_only_execute_tests_related_to_multiple_local_changes_in_production_code_when_affected_is_enabled() throws Exception {
+    public void should_only_execute_tests_related_to_multiple_local_changes_in_production_code_when_affected_is_enabled()
+        throws Exception {
         // given
         project.configureSmartTesting()
-                    .executionOrder(AFFECTED)
-                    .inMode(SELECTING)
+            .executionOrder(AFFECTED)
+            .inMode(SELECTING)
```

Comment:
Funny that I personally don't really need autoformatter - IDE really let's you stick to it when you use proper autocompletion. Anyway - we have few options to mitigate this 

  * we stick to this inlining which makes it less readable
  * we introduce regions (in IDEA) so formatter would skip it

>  I would rather keep formatting specified in .editorconfig file

This kind of information is not kept in `.editorconfig` file really, these are rules for IDEs